docs/wiki/basic-tutorials/running.md

```diff
@@ -0,0 +1,210 @@
+
+![](./running-idea-configurations-empty.png)
+
+**2.** In the newly opened window, click the `+` button in the top left and select `Application`.
+
+**3.** Give a name for your configuration and select the module to use the classpath of (usually `yourproject.main`).
+
+**4.** Select your Main class. Use the `...` button to search for it or provide the fully qualified name. If it can not be found, you most likely selected the wrong module in step 3.
+
+**5.** Optional: Set command line arguments and environment variables. For the environment variables, use the button to the right of the input field for a more convenient input window.
+
+**6.** Click `Apply` to finalize the configuration, then `OK` to close the window.
+
+![](./running-idea-configurations-create.png)
+
+**7.** Select your configuration in the drop down menu and run or debug it with the buttons to the right.
+
+![](./running-idea-configurations-bar.png)
+
+### Eclipse
+
+This assumes your project is set up correctly, can be built without errors, and does not yet have any run/debug configurations.
+
+**1.** In the menu bar, click "Run" then "Run Configurations...".
+
+**2.** In the newly opened window, select "Java Application" on the left side, then click the leftmost button in the row above the tree view. A new configuration will appear.
+
+![](./running-eclipse-configurations-empty.png)
+
+**3.** Give a name to your configuration.
+
+**4.** Set the project and the main class. To easily select it, use the "Browse..." and "Search..." buttons.
+
+**5.** Optional: Set command line (and VM) arguments as well as environment variables in their respective tabs.
+
+**6.** Click `Apply` to save your configuration, then `Close` to close the window.
+
+![](./running-eclipse-configurations-create.png)
+
+**7.** Run or debug your bot via the Buttons in the top row, the `Run` menu, or the shortcuts Ctrl+F11 for running and F11 for debugging.
+
+![](./running-eclipse-configurations-bar.png)
+
+## :package: Deploying and Running as a Standalone Application
+
+Running from the IDE is only recommended during development and strongly discouraged for production use. Generally, you'll want your build tool to create a convenient distribution format for you to use.
+
+### Building a Distribution with Gradle
+
+For Gradle, only two further steps are necessary for a basic application. On top of the steps described in the [Getting Started Section](/wiki/getting-started/intellij-gradle), also add the [Application Plugin](https://docs.gradle.org/current/userguide/application_plugin.html) and define your `mainClassName` as the fully qualified name of your main class. Since the `application` plugin implicitly applies the `java` plugin, you may leave that one out.
+
+Your modified `build.gradle` file should now look like this.
+```groovy
+plugins {
+    id 'application'
+}
+
+group 'com.github.yourname'
+version '1.0-SNAPSHOT'
+
+sourceCompatibility = 1.8
```

Comment:
I think this should be
```groovy
java {
    sourceCompatibility = VERSION_1_8
}
```
since some time, having it on the top-level is deprecated afair.